### PR TITLE
Book - Document nonceable CLI subcommands

### LINKED ISSUE #7444: CLI - Using durable nonces for offline signing is undocumented
#### Problem

Supersedes #7084.

Augmenting transaction generation with a durable nonce to allow for offline signing needs to be documented. 

#### Proposed Solution

As #7121 and #7419 come in. Be sure to document them

### PR DESCRIPTION
**Pull request by t-nelson**
#### Problem

CLI subcommands that support durable nonces are undocumented

#### Summary of Changes

Bump CLI API docs
Add documentation and example

Fixes #7444

### CODE REVIEW

File: book/src/offline-signing/durable-nonce.md [hunk 1]
```diff
@@ -186,3 +189,91 @@ solana authorize-nonce-account nonce-keypair.json nonce-authority.json
...
+$ solana airdrop -k alice.json 10 SOL
+10 SOL
+```
+
+#### - Create Alice's nonce account
+
+Now Alice needs a nonce account. Create one
+
+```bash
+$ solana create-nonce-account -k alice.json nonce.json 1 SOL
```
Comment (jstarry) [inline book/src/offline-signing/durable-nonce.md]: I think it's worth adding a note that this implies the nonce authority is alice

File: book/src/offline-signing/durable-nonce.md [hunk 2]
```diff
@@ -186,3 +189,91 @@ solana authorize-nonce-account nonce-keypair.json nonce-authority.json
...
+blockhash stored there
+
+```bash
+$ solana show-nonce-account nonce.json 
+balance: 1 SOL
+minimum balance required: 0.00136416 SOL
+nonce: F7vmkY3DTaxfagttWjQweib42b6ZHADSx94Tw8gHx3W7
+```
+```bash
+$ solana pay -k alice.json --blockhash F7vmkY3DTaxfagttWjQweib42b6ZHADSx94Tw8gHx3W7 --nonce nonce.json bob.json 1 SOL
```
Comment (jstarry) [inline book/src/offline-signing/durable-nonce.md]: Again, here the authority is implicitly alice

File: book/src/offline-signing/durable-nonce.md [hunk 3]
```diff
@@ -186,3 +189,91 @@ solana authorize-nonce-account nonce-keypair.json nonce-authority.json
...
+#### - Nonce to the rescue!
+
+Alice retries the transaction.  This time specifying her nonce account and the
+blockhash stored there
+
+```bash
+$ solana show-nonce-account nonce.json 
+balance: 1 SOL
+minimum balance required: 0.00136416 SOL
+nonce: F7vmkY3DTaxfagttWjQweib42b6ZHADSx94Tw8gHx3W7
```
Comment (jstarry) [inline book/src/offline-signing/durable-nonce.md]: Aside, but maybe we should print the nonce authority pubkey as well

File: book/src/offline-signing/durable-nonce.md (hunk 1 above)
Comment (t-nelson) [reply book/src/offline-signing/durable-nonce.md]: Good point! I'll actually spruce that up throughout

File: book/src/offline-signing/durable-nonce.md (hunk 3 above)
Comment (t-nelson) [reply book/src/offline-signing/durable-nonce.md]: I had that thought as well while copy/pasting this text 🙂

File: book/src/offline-signing/durable-nonce.md (hunk 2 above)
Comment (jstarry) [reply book/src/offline-signing/durable-nonce.md]: In this case, the nonce authority is implicitly alice: https://github.com/solana-labs/solana/blob/master/cli/src/cli.rs#L931

File: book/src/offline-signing/durable-nonce.md [hunk 4]
```diff
@@ -186,3 +190,100 @@ solana authorize-nonce-account nonce-keypair.json nonce-authority.json
...
+$ solana airdrop -k alice.json 10 SOL
+10 SOL
+```
+
+#### - Create Alice's nonce account
+
+Now Alice needs a nonce account. Create one
+
+{% hint style="info" %}
+Here, no separate [nonce authority](#nonce-authority) employed, so `alice.json`
```
Comment (CriesofCarrots) [inline book/src/offline-signing/durable-nonce.md]: "is employed"?

**Pull request merged**